source/admin_manual/clients.rst

```diff
@@ -0,0 +1,233 @@
+===============================
+Client issues and configuration
+===============================
```

Comment:
I would put a note here saying this is an old page, which went out of date at one point or another in the past. There must be several other clients out there, that are not described here, and I guess some that are here, are not in use anymore as well.